tests/integration/integration_test.go

```diff
@@ -840,3 +865,11 @@ func (ai *AnvilInstance) setBalance(address common.Address, balance *big.Int) er
 func (ai *AnvilInstance) mine(blockCount, timestampInterval *big.Int) error {
 	return ai.WsClient.Client.Client().Call(nil, "anvil_mine", "0x"+blockCount.Text(16), "0x"+timestampInterval.Text(16))
 }
+
+func (ai *AnvilInstance) getCurrentBlockHeight() (*big.Int, error) {
+	var result hexutil.Big
+	if err := ai.WsClient.Client.Client().Call(&result, "eth_blockNumber"); err != nil {
```

Comment:
You could simply do `ai.WsClient.BlockNumber()` here - though this returns a `uint64`, so there are some more places to change this.

Btw, the only reason an RPC call was made manually above is because `anvil_mine` is not standard, it's specific from anvil.

Reply:
True, removed